// Copyright (c) 2016-present, Facebook, Inc.
// All rights reserved.
//
// This source code is licensed under the BSD-style license found in the
// LICENSE file in the root directory of this source tree. An additional grant
// of patent rights can be found in the PATENTS file in the same directory.
syntax = "proto3";

package magma.orc8r;
option go_package = "magma/orc8r/cloud/go/protos";

message RedisState {
    bytes serialized_msg = 1;
    uint64 version = 2;
    bool is_garbage = 3;
}

